ceph-dev-new-build/config/definitions/ceph-dev-new-build.yml

```diff
@@ -105,6 +106,13 @@
             - ../../../scripts/build_utils.sh
             - ../../build/setup_osc
             - ../../build/build_osc
+      # mingw build scripts (targeting Windows)
+      - shell:
+          !include-raw:
+            - ../../build/validate_mingw
```

Comment:
Actually, if I am understanding the jenkins build process correctly, these scripts are executed on all builders for all distro types. Normally, the `validate_XYZ` scripts will `exit 1` to indicate that it shouldn't build on this host. Therefore, it seems like on an Ubuntu host, for example, it will first run the `build_deb` script above and then come down here to the `build_mingw` script. Would the `validate_XYZ` scripts need to be tweaked to also ensure that when `DIST == windows`, it's not building DEBs and when `DIST != windows` it's not building Windows here?

Reply:
Yep, the `DIST` variable should also be evaluated.

Reply:
`validate_deb` should also be modified

Reply:
@djgalloway I modified and force-pushed. Look good to you?